src/components/header/header.css

```diff
@@ -0,0 +1,50 @@
+.nav,
+.nav__container__parnav {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #c0c0c0;
+  width: 100vw;
+}
+
+.nav__container__parnav__brand {
+  display: flex;
+  align-items: center;
+}
+
+.nav__container__parnav__brand__logo {
+  display: flex;
+  align-items: center;
+}
```

Comment:
[OPTIONAL] Kindly ensure that your design matches that of the given design. The h3 text of the logo has a different color than the one assigned currently. Please check and rectify this. :pray: 

### Your design

![Screenshot from 2022-12-23 16-21-04](https://user-images.githubusercontent.com/65233860/209366961-150409b0-94cb-43d5-8e2c-c0603467b10f.png)


### The actual design

![Screenshot from 2022-12-23 16-21-22](https://user-images.githubusercontent.com/65233860/209366979-7908c452-2d2d-4250-b032-4d16ec8e5120.png)